v0/sources/shopify/config.js

```diff
@@ -10,12 +10,30 @@ const IDENTIFY_TOPICS = {
 // Mapping from shopify_topic name for ecom events
 const ECOM_TOPICS = {
   CHECKOUTS_CREATE: "checkouts_create",
-  ORDERS_UPDATED: "orders_updated"
+  CHECKOUTS_UPDATE: "checkouts_update",
+  ORDERS_UPDATE: "orders_updated",
+  ORDERS_CREATE: "orders_create"
 };
 
 const RUDDER_ECOM_MAP = {
   checkouts_create: "Checkout Started",
-  orders_updated: "Order Updated"
+  checkouts_update: "Checkout Updated",
+  orders_updated: "Order Updated",
+  orders_create: "Order Created"
+};
+
+const SHOPIFY_TRACK_MAP = {
+  checkout_delete: "Checkout Deleted",
+  carts_create: "Cart Create",
+  carts_update: "Cart Update",
+  fulfillments_create: "Fulfillments Create",
+  fulfillments_update: "Fulfillments Update",
+  orders_delete: "Order Deleted",
+  orders_edited: "Order Edited",
+  orders_cancelled: "Order Cancelled",
+  orders_fulfilled: "Order Fulfilled",
+  orders_paid: "Order Paid",
+  orders_partially_fullfilled: "Order Partially Fullfilled"
```

Comment:
```suggestion
  orders_partially_fullfilled: "Order Partially Fullfilled"
```
  orders_partially_fulfilled: "Order Partially Fulfilled"

Reply:
We are not immediately changing these names as it would be breaking change kept the scope for future for graceful onboarding